stable/postgresql/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 7.6.0
+version: 7.6.1
```

Comment:
Please bump minor version since it's new feature

Reply:
+1

Reply:
Done